docs/reference/search/search-your-data/search-using-query-rules.asciidoc

```diff
@@ -105,8 +107,8 @@ You can add query rules using the <<put-query-ruleset>> call. This adds a rulese
 
 The following command will create a query ruleset called `my-ruleset` with two pinned document rules:
 
-* The first rule will generate a <<query-dsl-pinned-query>> pinning the <<mapping-id-field,`_id`>>s `id1` and `id2` when the `user.query` metadata value is a fuzzy match to either `puggles` or `pugs` _and_ the user's location is in the US.
-* The second rule will generate a <<query-dsl-pinned-query>> pinning the <<mapping-id-field, `_id`>> of `id3` specifically from the `my-index-000001` index and `id4` from the `my-index-000002` index when the `user.query` metadata value matches `beagles`.
+* The first rule will generate a <<query-dsl-pinned-query>> pinning the <<mapping-id-field,`_id`>>s `id1` and `id2` when the `user_query` metadata value is a fuzzy match to either `puggles` or `pugs` _and_ the user's location is in the US.
```

Comment:
@kderusso I don't see `user_query` anywhere in the example that follows. Maybe it's my ignorance of query rules - is it an implicit metadata field? Or did you mean to refer to [query_string](https://github.com/elastic/elasticsearch/pull/98605/files#diff-2e010660c0de11cc160e993131cd540e8620696a9ab2e10deb3fd2b8e0162f8aR132)?

Reply:
Good catch, changing to `query_string`